### PR TITLE
Allow account unlock via password reset

### PR DESCRIPTION
**Pull request by robotdan**
https://github.com/FusionAuth/fusionauth-issues/issues/383

### CODE REVIEW

File: src/fa-screenshot.sh [hunk 1]
```diff
@@ -150,7 +150,7 @@ tell application theApp
 	activate
 	reopen
 	set xAxis to $xAxis
-	set yAxis to 360
+	set yAxis to 200
```
Comment (spwitt) [inline src/fa-screenshot.sh:153]: Is this a universal change that applies to all screenshots?

File: site/docs/v1/tech/apis/_tenant-request-body.adoc [hunk 2]
```diff
@@ -365,13 +365,21 @@ include::../shared/_advanced-edition-blurb-api.adoc[]
 +
 :advanced_feature!:
 
+[field]#tenant.failedAuthenticationConfiguration.actionCancelPolicy.onPasswordReset# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.42.0#::
+Indicates if you want the user to be able to self-service unlock their account prior to the action duration by completing a password reset workflow.
```
Comment (spwitt) [inline site/docs/v1/tech/apis/_tenant-request-body.adoc:369]:
```suggestion
Indicates whether you want the user to be able to self-service unlock their account prior to the action duration by completing a password reset workflow.
```

File: site/docs/v1/tech/apis/_tenant-request-body.adoc [hunk 3]
```diff
@@ -365,13 +365,21 @@ include::../shared/_advanced-edition-blurb-api.adoc[]
...
+
 [field]#tenant.failedAuthenticationConfiguration.actionDuration# [type]#[Long]# [optional]#Optional# [default]#defaults to `3`# [since]#Available since 1.8.0#::
 The duration of the User Action. This value along with the `actionDurationUnit` will be used to set the duration of the User Action. Value must be greater than 0.
 
 [field]#tenant.failedAuthenticationConfiguration.actionDurationUnit# [type]#[String]# [optional]#Optional# [default]#defaults to `"MINUTES"`# [since]#Available since 1.8.0#::
 
 include::_expiry_unit.adoc[]
 
+[field]#tenant.failedAuthenticationConfiguration.emailUser# [type]#[String]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.42.0#::
+Indicates you would like to email the user when the user's account is locked due to this action being taken. This requires the User Action specified by the [field]#tenant.failedAuthenticationConfiguration.userActionId# also to be configured for email. If the User Action is not configured to be able to email the user, this configuration will be ignored. See [field]#userEmailingEnabled# on the User Action.
```
Comment (spwitt) [inline site/docs/v1/tech/apis/_tenant-request-body.adoc:379]:
```suggestion
Indicates you would like to email the user when the user's account is locked due to this action being taken. This requires the User Action specified by the [field]#tenant.failedAuthenticationConfiguration.userActionId# to also be configured for email. If the User Action is not configured to be able to email the user, this configuration will be ignored. See [field]#userEmailingEnabled# on the User Action.
```

File: src/fa-screenshot.sh (hunk 1 above)
Comment (robotdan) [reply src/fa-screenshot.sh:153]: This adjusts how the script positions the safari window on your monitor. It doesn't affect the screenshot size.

File: site/docs/v1/tech/apis/_tenant-request-body.adoc (hunk 2 above)
Comment (robotdan) [reply site/docs/v1/tech/apis/_tenant-request-body.adoc:369]: Missed this before I merged. Will make this change thanks!

File: site/docs/v1/tech/apis/_tenant-request-body.adoc (hunk 3 above)
Comment (robotdan) [reply site/docs/v1/tech/apis/_tenant-request-body.adoc:379]: Missed this before I merged. Will make this change thanks!

**Pull request merged**